mmeds/tools/picrust2.py

```diff
@@ -0,0 +1,51 @@
+from mmeds.config import DATABASE_DIR
+from mmeds.util import get_file_index_entry_location
+from mmeds.tools.tool import Tool
+
+
+class PiCRUSt2(Tool):
+    """ A class for picrust2 analysis of uploaded studies. """
+
+    def __init__(self, queue, owner, access_code, parent_code, tool_type, analysis_type, config, testing, runs,
+                 run_on_node, analysis=True, restart_stage=0, kill_stage=-1, child=False):
+        super().__init__(queue, owner, access_code, parent_code, tool_type, analysis_type, config, testing, runs,
+                         run_on_node=run_on_node, analysis=analysis, restart_stage=restart_stage, kill_stage=kill_stage,
+                         child=child)
+        if testing:
+            load = 'module use {}/.modules/modulefiles; module load picrust1;'.format(DATABASE_DIR.parent)
```

Comment:
what does `module use` do? haven't seen that before

Reply:
not sure, good catch. I didn't update the testings of these because these don't actually get run at the moment. will fix